csrc/lower_validation.cpp

```diff
@@ -844,21 +844,6 @@ void validatePartialSplit(Fusion* fusion) {
 
 namespace {
 
-//! Utility to make sure targeted gpu capability is
-//!  higher than provided major.minor.
-void validateMinimumArch(int major, int minor) {
-  // Skip checking arch if disabled.
-  if (isOptionDisabled(DisableOption::ArchCheck)) {
```

Comment:
Is this option still used?

Reply:
Yes, in `executor_utils.cpp` `getCompiledKernel`, there is a
```
  TORCH_CHECK(
      !isOptionDisabled(DisableOption::ArchCheck),
      "NVFuser Compile: arch check disabled, should not return any compiled kernel");
```
don't think this makes much sense either.... Let me remove it.